client/src/views/CreateList/index.js

```diff
@@ -44,26 +43,32 @@ export class CreateList extends React.Component {
                 rows={5}
                 cols={30}
                 value={this.state.description}
-                onChange={e => this.setState({ description: e.target.value })}
+                onChange={this.handleSetState('description')}
                 autoResize
               />
             </label>
 
             <div>
-              <Button
-                type="button"
-                className="p-my-5 p-button-rounded p-mr-6"
-                label="Add Place"
-              />
               <Button
                 type="submit"
-                className="p-my-5 p-button-success p-button-rounded"
+                className="p-my-5 p-button-success p-button-rounded p-mr-6"
                 label="Create List"
               />
+              <Button
+                type="button"
+                className="p-my-5 p-button-rounded "
+                label="Add Place"
+              />
             </div>
           </div>
         </div>
       </form>
     );
   }
 }
+
+const mapDispatch = {
+  createList,
+};
+
+export const CreateList = connect(null, mapDispatch)(CreateListClass);
```

Comment:
Dumb component should not have any connect. Pass the props to run the connected functions from the parent instead.